server/src/game/event/on_role_switch.rs

```diff
@@ -0,0 +1,18 @@
+use crate::game::{player::PlayerReference, Game};
+
+pub struct OnRoleSwitch{
```

Comment:
Should be renamed to BeforeRoleSwitch or AfterRoleSwitch. I have no clue when this is called